services/crosschainconnector/ethereum/service.go

```diff
@@ -2,16 +2,71 @@ package ethereum
 
 import (
 	"context"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/orbs-network/orbs-network-go/instrumentation/log"
+	"github.com/orbs-network/orbs-network-go/services/crosschainconnector/ethereum/adapter"
 	"github.com/orbs-network/orbs-spec/types/go/services"
 )
 
+type ethereumConnectorConfig interface {
+	EthereumEndpoint() string
```

Comment:
It seems to me that the endpoint is a configuration of the adapter, not the service

I think the service has no config

Reply:
yeah, and i fixed the naming, but we cannot just configure a adapter, the configution injection happens at service level, or am i missing something?

Reply:
nvm, its done